pages/_app.js

```diff
@@ -26,6 +27,9 @@ export default function MyApp({ Component, pageProps, router }) {
 
   return (
     <AppContext.Provider value={{ path: router.asPath }}>
+      <Helmet>
+          <html lang="en" />
+        </Helmet>
```

Comment:
You cannot make this in this way. `next/Head` works similar to helmet so you should add that lang inside `Head`. Try this https://github.com/vercel/next.js/issues/9160#issuecomment-643867740 Also `Head` component has as parent `html` tag.

Reply:
Okay, I actually did try putting it in the `head` but didn't work out, will do it again following the reference you gave. Thanks. I actually removed the unnecessary codes earlier but forgot to save the file before committing. my bad.